### PR TITLE
Add SaleMove documentation links

### PR DESCRIPTION
**Pull request by ostap0207**
Also improve documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+Operator in the case Visitor needs more advanced human help.
+
+To get more information about SaleMove and our API visit:
+
+* SaleMove web site - https://salemove.com
+* JS API Documentation - https://js-sdk-docs.salemove.com
+* REST API Documentation - http://developer.salemove.com
+
+
+To get information on how to get access to the platform and require access keys, please, contact your account manager.
```
Comment (willmore) [inline README.md]: If you intend to only host this example on your local dev machine, the account manager will provide you a unique localhost name with format `<unique-id>.local.dev`. This hostname will be used to access and run this example application. See `Starting for local development` below for an example using nginx.

If you already have a hostname that resolves to your local development machine or target deployment machine, please provide that to the account manager.

**Comment by ostap0207** [comment]
Updated

**Comment by ostap0207** [comment]
Will merge, any other comments can be addressed later

**Comment by ostap0207** [comment]
!merge

**Pull request merged**